examples/sensor/sensor/file_sensor_example.py

```diff
@@ -0,0 +1,49 @@
+# %% [markdown]
+# # File Sensor
+#
+# This example shows how to use the `FileSensor` to detect files appearing in your local or remote filesystem.
+#
+# To begin, import the required libraries.
+
+# %%
+from flytekit import task, workflow
+from flytekit.sensor.file_sensor import FileSensor
+
+# %% [markdown]
+# Create a FileSensor task.
```

Comment:
I recommend changing this to "Next, create a FileSensor task" and moving the next code block directly under this sentence.